lib/PlaidClient.js

```diff
@@ -210,6 +211,37 @@ Client.prototype.getTransactions =
     }, options, cb);
   };
 
+// getAllTransactions(String, Date, Date, Object?, Function)
+Client.prototype.getAllTransactions =
+  function(access_token, start_date, end_date, cb) {
+    var transactions = [];
+    var self = this;
+
+    P.coroutine(function*() {
+      while (true) {
```

Comment:
makes sense, we will break out when we get the correct number of transactions or if any of the requests throws an error.

assuming the plaid API works correctly, this looks right to me.